### DISCUSSION TITLE
domanda (forse) banale su automazione in assenza di internet

### DISCUSSION THREAD

**Discussion by jard159** [indomus/forum#1653] [HUB: Home Assistant]
Per i più esperti sembrerà una domanda banale ma... è sbagliata la mia configurazione o quando si verifica una assenza temporanea di connessione ad internet (ma lan perfettamente funzionante), qualsiasi automazione fallisce completamente nell'esecuzione quando incontra una notifica mail (notify-smtp) anche se poi subito dopo ci sono notifiche standard (persistent_notification) o altre azioni (mqtt) che possono tranquillamente essere eseguite in locale??? ho notato che mi vanno in errore tutte le automazioni che presentano uno scenario del genere....

c'è una soluzione a una condizione di questo tipo???
qualcosa del tipo... "se anche si verifica un errore timeout (ad esempio) vai avanti comunque???

grazie mille

**Comment by snakuzzo** [comment]
Non ho capito se alla fine funzionano oppure no.
Da quello che ho capito fallisce questo notify smtp che, dal nome, immagino che tenti la connessione ad un server smtp su internet per un invio mail. In tal caso è normale che fallisca se non c'è internet, ma le altre azioni dovrebbero continuare ad essere eseguite.

**Reply by jard159** [reply]
Ciao, purtroppo dalle mie prove niente da fare.... l'automazione non va avanti....
Postando la domanda sul forum HA ne è uscita fuori una discussione che ha portato gli sviluppatori ad implementare una sorta di "prosegui anche se trovi errori nell'esecuzione" nella prossima release.

Di seguito il link:  https://community.home-assistant.io/t/add-option-to-automation-for-how-to-respond-to-exception/408199

**Reply by snakuzzo** [reply]
Ah perfetto. Buono a sapersi. Grazie

**Reply by jard159** [reply]
grazie sempre a voi!!!!!
